contrib/pai_vscode/README.md

```diff
@@ -62,35 +65,40 @@ If there are multiple OpenPAI clusters, you can follow above steps again to conn
 
 ## Submit job
 
+There is a tutorial for OpenPAI job submission, please refer to [Submit job to OpenPAI](documentation/submit_job.md).
 After added a cluster configuration, you can find the cluster in *PAI CLUSTER EXPLORER* pane as below.
 
 ![pai cluster explorer](https://raw.githubusercontent.com/Microsoft/pai/master/contrib/pai_vscode/assets/pai_cluster_explorer.png)
 
-Submit V1 job:
+Submit V2 job (For OpenPAI v0.13.0 and above):
 
-You can create a job configuration and submit to OpenPAI as below steps.
+You can create a job v2 configuration and submit to OpenPAI as below steps.
 
-1. Make sure the value of `protocol_version` property in cluster configuration is `'1'`. Double click `Create Job Config...` in OpenPAI cluster Explorer, and then specify file name and location to create a job configuration file.
-2. Update job configuration as needed. If you are not familiar with this configuration file, learn from [here](https://github.com/Microsoft/pai/blob/master/docs/user/training.md#learn-hello-world-job).
-3. Right click on the created job configuration file, then click on `Submit Job to PAI Cluster`. The client will upload files to OpenPAI and create a job. Once it's done, there is a notification at right bottom corner, you can click to open the job detail page.
+1. Create job config file:
+    1. Double click `Create Job Config...` in OpenPAI cluster Explorer, and then specify file name and location to create a job configuration file (Make sure the value of `protocol_version` property in cluster configuration is `'2'`).
+    2. Right click python or cntk file in VSCode Explorer and select `Create PAI Job Config V2`, and then specify file name and location to create a job configuration file.
+2. Update job configuration as needed. If you are not familiar with this configuration file, learn from [here](https://github.com/microsoft/pai/blob/master/docs/marketplace-and-submit-job-v2/marketplace-and-submit-job-v2.md#introduction-to-yaml-file).
+3. Right click on the created job v2 configuration file, then click on `Submit Job to PAI Cluster`. The client will upload files to OpenPAI and create a job. Once it's done, there is a notification at right bottom corner, you can click to open the job detail page.
 
     If there are multiple OpenPAI clusters, you need to choose one.
 
     This animation shows above steps.
-    ![submit job](https://raw.githubusercontent.com/Microsoft/pai/master/contrib/pai_vscode/assets/submit-job.gif)
+    ![submit job](https://raw.githubusercontent.com/Microsoft/pai/master/contrib/pai_vscode/assets/submit-job-v2.gif)
 
-Submit V2 job:
+Submit V1 job:
```

Comment:
Submit OpenPAI V1 Job (deprecating, only for OpenPAI version under 0.13.0)

Reply:
Done.